src/visualization/utils/getFormatter.test.ts

```diff
@@ -0,0 +1,113 @@
+import {getFormatter} from './getFormatter'
+import {DEFAULT_TIME_FORMAT} from '../../shared/constants'
+import {Base, TimeZone} from '../../types'
+
+// July 4th, 1983, 20:00:00 UTC
+const timestamp = 426196800000
+
+describe('getFormatter', () => {
+  it("should format correctly for base 10 and 'number' columntype", function() {
+    const formatterOptions = {
+      prefix: 'prefix ',
+      suffix: ' suffix',
+      base: '10' as Base,
+      format: true,
+    }
+
+    const formatter = getFormatter('number', formatterOptions)
+
+    const formattedValue = formatter(100000)
+
+    expect(formattedValue).toBe('prefix 100k suffix')
+  })
+
+  it("should format correctly for base '' 'number' columntype", function() {
+    const formatterOptions = {
```

Comment:
I also added tests for the other part of the getFormatter, which isn't connected to this PR's fix (time related formatting), so if we want to add those tests as a separate PR I can do that!

Reply:
nice!